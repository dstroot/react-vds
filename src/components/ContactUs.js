import React from 'react';
import { NavLink } from 'react-router-dom';
// import SVG files directly as React components. The ReactComponent import
// name is special and tells Create React App that you want a React
// component that renders an SVG, rather than its filename.
import { ReactComponent as Robot } from '../media/robot.svg';

const ContactUs = () => {
  return (
    <div className="position-relative overflow-hidden p-2 p-md-5 m-md-3 text-center">
      <div className="col-md-7 p-lg-2 mx-auto my-5">
        <h1 className="display-4 text-primary">Get in Touch</h1>
        <p className="lead">
          Verite Data Science is providing customized solutions for both
          startups and Fortune 100 companies. Get in touch to see how we can
          help.
        </p>
        <div className="svg-container mx-auto">
          <Robot />
        </div>
        <br />
        <NavLink className="btn btn-lg btn-outline-secondary" to="/contact">
          Contact Us
        </NavLink>
      </div>
    </div>
  );
};

export default ContactUs;
