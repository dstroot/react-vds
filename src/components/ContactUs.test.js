import React from 'react';
import ContactUs from './ContactUs';
import renderer from 'react-test-renderer';

test('ContactUs renders some content', () => {
  const component = renderer.create(<ContactUs />);

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
