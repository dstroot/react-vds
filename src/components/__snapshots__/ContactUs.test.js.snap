// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactUs renders some content 1`] = `
<div>
  <div
    className="position-relative overflow-hidden p-2 p-md-5 m-md-3 text-center bd-text-purple-bright"
  >
    <div
      className="col-md-5 p-lg-5 mx-auto my-5"
    >
      <h1
        className="display-4"
      >
        Punny headline
      </h1>
      <p
        className="lead"
      >
        And an even wittier subheading to boot. Jumpstart your marketing efforts with this example based on Apple's marketing pages.
      </p>
      <a
        className="btn btn-lg btn-outline-secondary"
        href="/"
      >
        Coming soon
      </a>
    </div>
  </div>
</div>
`;
